CHANGELOG.md

```diff
@@ -6,13 +6,23 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 ## [Unreleased]
 
+### Changed
+
+- Minor changes to what is shown in the data checker
+- Duplicate titles are now allowed in the same session
+- Pre-entry players no longer play a part in grist gutter capacity,
+and they no longer receive grist from the gutter
+
 ### Fixed
 
 - Metal Boats no longer crash the game when dispensed
+- Fixes some niche issues with connections
+- `/sburbpredefine` now gives an error when applied to a player that already has a land
```

Comment:
It's not a runtime exception that might crash the game, but a command exception, a special type of exception that aborts the command and gives a feedback message to the command executor.

Previously for players with a land, the command would predefine data as normal. That was bad because that data would then never actually be applied to the player (because their land and title has already been set), even if the command reported it as a success.